asab/storage/elasticsearch.py

```diff
@@ -851,3 +856,21 @@ def serialize(v):
 		return v.timestamp()
 	else:
 		return v
+
+
+def parse_url_config(url):
+	scheme = urllib.parse.urlparse(url).scheme
+	pattern = r'(https?://)([^:/]+):(\d+)/?'
```

Comment:
PLease use `urlib` for parsing, don't use regexes.

Reply:
As we have different formats urllib will parse the string differently - sometimes "the remainder" is going to be in netloc (what we have now),  but for new configurations it will be in path. So it won't be neat. Regex will be needed anyway.

Reply:
This won't work for all cases anymore

https://github.com/TeskaLabs/asab/blob/master/asab/storage/elasticsearch.py#L59-L62